frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte

```diff
@@ -113,14 +115,15 @@
   </div>
   <div class="mt-6">
     <!-- TODO: Add project count once orgs can own projects -->
-    <OrgTabs bind:activeTab={$queryParamValues.tab} memberCount={org.members.length} projectCount={0} />
+    <OrgTabs bind:activeTab={$queryParamValues.tab} hideSettingsTab={!canSeeSettings} memberCount={org.members.length} projectCount={0} />
   </div>
   <div class="py-6 px-2">
     {#if $queryParamValues.tab === 'projects'}
     Projects list will go here once orgs have projects associated with them
     {:else if $queryParamValues.tab === 'members'}
     <OrgMemberTable
       shownUsers={org.members}
+      showEmailColumn={user.isAdmin}
```

Comment:
With `.AdminRequired` on the `User.Email` field, the only way for a DTO that bypasses AdminRequired to exist is if we write our own resolver, right? That's what you're referring to here: a new OrgMemberDto class, whose OrgMemberDtoGqlConfiguration would create a custom resolver that loads data from the User entity without going through the GQL resolver for User (since that would activate the AdminRequired policy).

... Or, wait, the MeDto class didn't need a custom resolver. I might be overthinking this.

Reply:
Org admins are now able to see email addresses (and usernames) in the GraphQL query, so I've changed this check to `{canManage}` instead of `{user.isAdmin}`. The `canManage` check allows site admins *and* org admins. So this is fully resolved now.